tools/download.sh

```diff
@@ -20,52 +19,52 @@ case "$OSTYPE" in
         # Install curl and wget to download package
         case "$ID" in
             centos*)
-                if [[ -z $(command -v curl) ]]; then
+                if ! hash curl 2>/dev/null; then
                     echo "curl not found, installing..."
                     sudo yum install -y curl
                 fi
-                version=rpm
+
+                package=powershell-6.0.0_alpha.9-1.el7.centos.x86_64.rpm
                 ;;
             ubuntu)
+                if ! hash curl 2>/dev/null; then
+                    echo "curl not found, installing..."
+                    sudo apt-get install -y curl
+                fi
+
                 case "$VERSION_ID" in
                     14.04)
-                        version=ubuntu1.14.04.1_amd64.deb
+                        package=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
                         ;;
                     16.04)
-                        version=ubuntu1.16.04.1_amd64.deb
+                        package=powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
```

Comment:
I wouldn't. The versions used in the packages all different slightly due to package naming schema. This has already bit us before where someone assumed the version for the RPM was the same for the DEB, but of course they're not. And they're not the same as the tag above either. The $release variable is honestly a mistake; I did it for readability but these links (along with those in the readme) should all be updated programmatically with xargs sed.